template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/AbstractRoleConfigConfigProvider.java

```diff
@@ -97,7 +97,7 @@ private boolean isRefNameAndTypeMatch(String roleType, String roleConfigGroupsRe
 
     protected abstract List<ApiClusterTemplateConfig> getRoleConfig(String roleType, HostgroupView hostGroupView, TemplatePreparationObject source);
 
-    protected List<ApiClusterTemplateVariable> getVariables(String roleType, HostgroupView hostgroupView, TemplatePreparationObject source) {
+    protected List<ApiClusterTemplateVariable> getRoleConfigVariable(String roleType, HostgroupView hostgroupView, TemplatePreparationObject source) {
```

Comment:
I would prefer plural form (for both methods), since they return lists.

Also, variable assignments are not specific to a role config (they end up in the `variables` section of the template), so I would prefer to keep `getVariables` and rename `getRoleConfigVariables` (in the interface) instead.

Reply:
Thanks for the suggestions. This will be considered in a separate PR after an agreement is made regarding the exact naming. 😃